### DISCUSSION TITLE
Status schema updates

### DISCUSSION THREAD

**Discussion by JohannaOtt** [INSPIRE-MIF/helpdesk#184] [Q&A] [answered]
Hi,

I have two questions related to the INSPIRE 5.0 schemas:

1. Are all 5.0 schemas released by now or do you expect more schemas to follow in the next weeks and months?
2. Are the 5.0 schemas legally binding for the data published from a specific point in time or can the data still be provided using the 4.0 schemas?

Thank you and all the best

Johanna

**Comment by JohannaOtt** [comment]
@fabiovinci Who could help us with the two questions raised?

**Comment by fabiovinci** [comment, marked as answer]
Dear @JohannaOtt,

I'm sorry for not getting back to you sooner.

Regarding your question:

1. All the schemas should now be considered final. Due to the many changes introduced and dependencies between schemas, there were several hotfix releases for the 2024.1 release, but it was an exceptional case. The next release will be 2024.2, due by July 31, 2024.
2. According to the current rules, i.e. `Deprecated versions should no longer be used after 2 years after a major release`, version 4 can still be used. Please note that MIG/MIG-T will discuss these rules in the coming months; updates may follow.

I hope this helps clear things up.

**Comment by JohannaOtt** [comment]
Hi Fabio,

Thank you for the information.

> Please note that MIG/MIG-T will discuss these rules in the coming months; updates may follow.

Ok, would be great if you could add it here once more input is available.


All the best

Johanna

**Comment by fabiovinci** [comment]
Hi @JohannaOtt,

please be advised to refer to the [XML Schemas](https://knowledge-base.inspire.ec.europa.eu/tools/xml-schemas_en) page located in the IKB for official references. The information is currently not up-to-date, but it will be updated soon.
